docs/design/integrate-with-cc-tools/README.md

```diff
@@ -0,0 +1,308 @@
+# FPC and CC-tools integration
+
+The goal of this project is to combine the strength of [cc-tools](https://github.com/hyperledger-labs/cc-tools) with the enhanced security capabilities of [Fabric Private Chaincode (FPC)](https://github.com/hyperledger/fabric-private-chaincode).
```

Comment:
I saw a couple of different spellings of "CC-tools", "cc-tools", "CC tools". Let's stick with one :)

Reply:
Done